app/models/booking.rb

```diff
@@ -5,9 +5,9 @@ class Booking < ApplicationRecord
 
   validate :end_date_after_start_date
 
-  validate :start_date_not_before_present
```

Comment:
I HAVE DONE THIS SO THAT WE CAN TEMPORARILY SEED SOME DATES IN THE PAST TO SHOW OUR REVIEW FEATURE. WE WILL CHANGE THIS BACK FOR THE DEMO